numpy/lib/ufunclike.pyi

```diff
@@ -0,0 +1,50 @@
+from typing import Any, overload, TypeVar, List, Union
+
+from numpy import floating, bool_, ndarray
+from numpy.typing import (
+    _ArrayLikeFloat_co,
+    _ArrayLikeObject_co,
+    _ArrayOrScalar,
+)
+
+_ArrayType = TypeVar("_ArrayType", bound=ndarray[Any, Any])
+
+__all__: List[str]
+
+@overload
+def fix(
+    x: _ArrayLikeFloat_co,
+    out: None = ...,
+) -> _ArrayOrScalar[floating[Any]]: ...
+@overload
```

Comment:
Be good to have 2 blank lines between functions (PEP8).

Reply:
I know you are trying to group the functions, but I find it more difficult to read without the blank lines.

Reply:
For overloads it's generally common practice to exclude the blank lines ([example from typeshed](https://github.com/python/typeshed/blob/8f7e2216b66b9e8181b119fa2f7d560fca987605/stdlib/builtins.pyi#L677-L680)).

Note that this area is not quite covered by PEP 8 though: the description of a single function with multiply signatures.